pkg/model/externalconfig.go

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright (c) 2023 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"crypto/sha256"
```

Comment:
that's what other do for exactly the same task, TBH I had no time to optimize it and do not think it will have big performance impact as it is called quite rarely, but I do not mind to replace it if you propose your option

Reply:
Huh, maybe sha256 is newer/more optimized now, carry on :)
```
The 'numbers' are in 1000s of bytes per second processed.
type             16 bytes     64 bytes    256 bytes   1024 bytes   8192 bytes  16384 bytes
md5             104574.53k   273858.63k   513204.40k   658448.10k   713978.01k   723875.91k
sha256          159849.35k   537076.76k  1377179.23k  2159443.74k  2600313.41k  2659953.40k
```